bitnami/etcd/Chart.yaml

```
@@ -25,4 +25,4 @@ name: etcd
 sources:
   - https://github.com/bitnami/bitnami-docker-etcd
   - https://coreos.com/etcd/
-version: 5.6.2
+version: 5.6.3
```

Bumping the chart version in a minor since it is a feature addition
```suggestion
version: 5.7.0
```